src/scan.c

```diff
@@ -1385,6 +1386,36 @@ void rotate_coords(int x, int y, int *xo, int *yo, int dxi, int dyi) {
 	}
 }
 
+void rotate_cursor_coords(int x, int y, int *xo, int *yo, int dxi, int dyi) {
```

Comment:
why another function and not rotate_coords()?

Reply:
`rotate_coords()` gives incorrect coordinates for these rotations:
- `-rotate 90y`
- `-rotate 270`
- `-rotate 90`

Reply:
Add this explanation as a comment above the function and we're good to go!

Reply:
added in https://github.com/LibVNC/x11vnc/pull/163/commits/08538dfee9674b61ba77fc8476bf434afc27a1ed